src/core/DoomInPlayCounter.ttslua

```diff
@@ -1,132 +1,105 @@
-MIN_VALUE = -99
-MAX_VALUE = 999
-
-function onload(saved_data)
-    light_mode = false
-    val = 0
-
-    if saved_data ~= "" then
-        local loaded_data = JSON.decode(saved_data)
-        light_mode = loaded_data[1]
-        val = loaded_data[2]
-    end
-
-    createAll()
-end
-
-function updateSave()
-    local data_to_save = {light_mode, val}
-    saved_data = JSON.encode(data_to_save)
-    self.script_state = saved_data
-end
-
-function createAll()
-    s_color = {0.5, 0.5, 0.5, 95}
-
-    if light_mode then
-        f_color = {1,1,1,95}
-    else
-        f_color = {0,0,0,100}
-    end
-
-
-
+-- Doom-in-Play Counter
+-- made by:         Chr1Z
+-- description:     counts the doom tokens in play periodically (excluding the agenda), ignores objects with specified tag
+information = {
+    version = "1.1",
+    last_updated = "12.11.2022"
+}
+
+-- common parameters
+local CAST_PARAMETERS        = {}
+CAST_PARAMETERS.direction    = { 0, 1, 0 }
+CAST_PARAMETERS.type         = 3
+CAST_PARAMETERS.max_distance = 0
+
+local ypos       = 1.6
```

Comment:
This could use a more descriptive name.  What's it used for?  Same with zone below